doc/classes/AnimationNodeBlendTree.xml

```diff
@@ -44,6 +44,12 @@
 				Returns the sub animation node with the specified [param name].
 			</description>
 		</method>
+		<method name="get_node_list" qualifiers="const">
+			<return type="StringName[]" />
+			<description>
+				Returns a list containing names of all nodes.
```

Comment:
Although I'm not a fan, as the descriptions were written with a bit of botched English, "sub animation nodes" is for consistency with the rest:
```suggestion
				Returns a list containing the names of all sub animation nodes in this blend tree.
```